RELEASING.rst

```diff
@@ -0,0 +1,61 @@
+Creating a release
+==================
+
+This document is documentation intended for Buildbot maintainers. It documents the release
```

Comment:
Buildbot style is one sentence per line

Reply:
Argh, I keep forgetting that it's not only for internal docs :-) Fixed now.